src/test/java/com/academia/andruhovich/library/util/JsonConvertHelper.java

```diff
@@ -0,0 +1,127 @@
+package com.academia.andruhovich.library.util;
+
+import com.academia.andruhovich.library.dto.AuthResponseDto;
+import com.academia.andruhovich.library.dto.AuthorDto;
+import com.academia.andruhovich.library.dto.BookDto;
+import com.academia.andruhovich.library.dto.OrderResponseDto;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.DeserializationFeature;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.SerializationFeature;
+import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule;
+import lombok.Data;
+
+import java.io.IOException;
+import java.time.ZoneId;
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+import java.util.TimeZone;
+
+import static com.academia.andruhovich.library.exception.ErrorMessages.CANNOT_READ_JSON_VALUE;
+import static com.academia.andruhovich.library.util.DateHelper.CURRENT_LOCATION;
+
+@Data
+public class JsonConvertHelper {
+
+    private static final String CONTENT = "content";
+
+    private final static ObjectMapper objectMapper = new ObjectMapper()
+            .registerModule(new JavaTimeModule())
+            .setTimeZone(TimeZone.getTimeZone(ZoneId.of(CURRENT_LOCATION)))
+            .configure(SerializationFeature.WRITE_DATES_WITH_ZONE_ID, true)
+            .configure(DeserializationFeature.ADJUST_DATES_TO_CONTEXT_TIME_ZONE, false)
+            .disable(SerializationFeature.WRITE_DATES_AS_TIMESTAMPS);
+
+    public static String convertToJsonString(final Object obj) throws IOException {
+        objectMapper.registerModule(new JavaTimeModule());
+        return objectMapper.writeValueAsString(obj);
+    }
+
+    public static AuthResponseDto getAuthResponseDto(String response) {
+        AuthResponseDto authResponseDto = new AuthResponseDto();
+        try {
+            JsonNode responseBody = objectMapper.readTree(response);
+            authResponseDto = objectMapper.convertValue(responseBody, new TypeReference<>() {
+            });
+        } catch (JsonProcessingException e) {
+            throw new RuntimeException(String.format(CANNOT_READ_JSON_VALUE, response));
+        }
+        return authResponseDto;
+    }
+
+    public static AuthorDto getAuthorDto(String response) {
+        AuthorDto authorDto = new AuthorDto();
+        try {
+            JsonNode responseBody = objectMapper.readTree(response);
+            authorDto = objectMapper.convertValue(responseBody, new TypeReference<>() {
+            });
+        } catch (JsonProcessingException e) {
+            throw new RuntimeException(String.format(CANNOT_READ_JSON_VALUE, response));
+        }
+        return authorDto;
+    }
+
+    public static List<AuthorDto> getAuthorDtos(String response) {
+        List<AuthorDto> authors = new ArrayList<>();
```

Comment:
remove = new ArrayList<>(); do it below too

Reply:
Deleted JsonConvertHelper class